test/client/spec/victory-bar/victory-bar.spec.js

```diff
@@ -166,7 +165,8 @@ describe("components/victory-bar", () => {
           ]}
         />
       );
-      const Labels = wrapper.find(VictoryLabel);
+      const Labels = wrapper.find("VictoryLabel");
+      expect(Labels).to.have.lengthOf(3);
```

Comment:
for some reason this test was previously not matching any nodes, so this fixes that + makes sure we match some nodes